README.md

```diff
@@ -10,11 +47,13 @@ You can download Noto from [Github](https://github.com/alialbaali/Noto/releases/
 
```

Comment:
Can you re-organize the Download section?

You can remove all the text, and just place images centered. For GitHub, you can use [this](https://github.com/logos)

You can also add F-Droid too.

Thanks!

Reply:
All good, I'd just need the f-droid link!